src/lib/parse/CompoundValueNode.cpp

```diff
@@ -67,16 +61,10 @@ CompoundValueNode::CompoundValueNode():
 * CompoundValueNode - constructor for toplevel 'node'
 */
 CompoundValueNode::CompoundValueNode(orion::ValueType _type):
-  name        ("toplevel"),
+  name        (""),
```

Comment:
As far as I know, a std::string is already initialized to the empty string, so this line can probably be removed